### DISCUSSION TITLE
[@types/react-mentions] Failed to resolve import "react-mentions/lib/utils"

### DISCUSSION THREAD

**Discussion by sumansaurabh00001** [DefinitelyTyped/DefinitelyTyped#67953] [Issues with a @types package]
[@types/react-mentions] not able import react-mentions/lib/utils in a vite project but getting error

Failed to resolve import "react-mentions/lib/utils"

[a comment by an automated account is not shown]

**Comment by activescott** [comment]
@suman379 Is your issue specific to vite? Could you possibly provide a simple repro?
